### PR TITLE
Filter Layout

### PR DESCRIPTION
**Pull request by christinayitingyang**
screen shot 2018-02-14 at 12 19 25 pm

### CODE REVIEW

**Comment by jholm117** [comment]
I think the button colors are a little extra. I think this is what we should go for:
image

File: recipe-finder.html [hunk 1]
```diff
@@ -9,11 +9,18 @@
...
-        <div class="toolbar">
-            <select name="filter-choice" id="filter options">
-                <option value="vegetarian">Vegetarian</option>
-            </select>
-        </div>
+
+        <div class="dropdown">
+            <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">Recipe Filter
+            <span class="caret"></span></button>
+            <ul class="dropdown-menu">
```
Comment (jholm117) [inline recipe-finder.html]: the list markup is probably unnecessary, i think a bunch of buttons inside a div will suffice

File: recipe-finder.html [hunk 2]
```diff
@@ -9,11 +9,18 @@
...
+              <li><a href="#" class="btn btn-sm btn-danger">Popular <span class="badge badge-secondary">5</span></a></li>
```
Comment (jholm117) [inline recipe-finder.html]: if you look  in the picture I sent they made the dropdown items by simply
`<button class="dropdown-item" type="button">Action</button>`

File: recipe-finder.html (hunk 1 above)
Comment (christinayitingyang) [reply recipe-finder.html]: screen shot 2018-02-14 at 12 41 06 pm
How about this?

Comment (jholm117) [reply recipe-finder.html]: that looks better. What are the badges for?

Comment (christinayitingyang) [reply recipe-finder.html]: Just so they know how many recipes we have. They make the filter look more legit.

File: finder-stylesheet.css [hunk 3]
```diff
@@ -30,4 +30,13 @@ body{
...
     float: right;
+}
+
+.btn{
+    margin: 5px;
+    padding-right: 10px;
+}
+
+.badge{
+    margin-left: 10px;
```
Comment (jholm117) [inline finder-stylesheet.css]: I think `float: right` would look better so they're all aligned.
image

File: finder-stylesheet.css [hunk 4]
```diff
@@ -30,4 +30,13 @@ body{
 .recipe-info-wrapper{
     background-color: purple;
     float: right;
+}
+
+.btn{
+    margin: 5px;
+    padding-right: 10px;
```
Comment (jholm117) [inline finder-stylesheet.css]: i don't think this padding is necessary

File: recipe-finder.html [hunk 5]
```diff
@@ -9,14 +9,21 @@
...
-            <select name="filter-choice" id="filter options">
-                <option value="vegetarian">Vegetarian</option>
-            </select>
-        </div>
+
+        <div class="dropdown">
+            <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">Recipe Filter
+            <span class="caret"></span></button>
+            <ul class="dropdown-menu">
+                <button class="dropdown-item" type="button">Popular <span class="badge badge-secondary">5</span></button>
```
Comment (jholm117) [inline recipe-finder.html]: No unnecessary spaces within Html tags i.e. should just be `>Popular<`, `>Asian<`, etc.

Comment (christinayitingyang) [reply recipe-finder.html]: Kk. Changed

**Pull request merged**